src/components/Settings/SettingsMain/index.tsx

```diff
@@ -45,11 +45,16 @@ const messages = defineMessages('components.Settings.SettingsMain', {
     'The "Process Blacklisted Tags" job will blacklist this many pages into each sort. Larger numbers will create a more accurate blacklist, but use more space.',
   streamingRegion: 'Streaming Region',
   streamingRegionTip: 'Show streaming sites by regional availability',
+  hideBlacklisted: 'Hide Blacklisted Items',
+  hideBlacklistedTip:
+    'Hide blacklisted items from the discover pages but not search results',
```

Comment:
Maybe here too? 🤔
Wdyt @gauthier-th 
To mention about this being admin specific

Reply:
I'm not sure mentioning it in the tip makes much sense, seeing as this page is only viewable by administrators anyways.

Reply:
> I'm not sure mentioning it in the tip makes much sense, seeing as this page is only viewable by administrators anyways. The actual feature applies for all users. 

Yeah but this setting is for admin. Not for all users. User blacklist views are handled by user permissions already.